app/workers/check_organisation_links_worker.rb

```diff
@@ -26,6 +26,6 @@ def public_editions(organisation)
   end
 
   def callback
-    Whitehall::UrlMaker.new(host: Plek.find('whitehall-admin')).admin_link_checker_api_callback_url
+    Whitehall::UrlMaker.new(host: Plek.new.external_url_for('whitehall-admin')).admin_link_checker_api_callback_url
```

Comment:
Could perhaps change AdminRequest in routes.rb to allow internal URLs as well?

Reply:
Yes, I think this is neater, I've done this in 30675a2